tests/test_unit/approval_client/client.py

```diff
@@ -481,30 +480,26 @@ def upsert(*args, **kwargs) -> Any:
                     upserted.extend(value)
```

Comment:
This is the mocking of the ToolkitClient (subclass of CogniteClient). We know return a list of object instead of an individual one.